app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt

```diff
@@ -45,26 +45,29 @@ fun PlayerControls(
 
         Row(
             modifier = Modifier.align(Alignment.Center),
-            horizontalArrangement = Arrangement.spacedBy(2.dp),
+            horizontalArrangement = Arrangement.spacedBy(40.dp),
             verticalAlignment = Alignment.CenterVertically
         ) {
             IconButton(onClick = onSkipPrevious) {
                 Icon(
+                    modifier = Modifier.size(30.dp),
                     imageVector = Icons.Default.SkipPrevious,
                     contentDescription = stringResource(R.string.skip_previous)
                 )
             }
 
-            IconButton(onClick = onClickPlayPause) {
+            IconButton(
+                modifier = Modifier.size(64.dp),
+                onClick = onClickPlayPause) {
```

Comment:
This `) {` should be carried on to the next line like how it is everywhere else

Reply:
bet